docker.go

```diff
@@ -752,6 +754,11 @@ func readTCPropsFile() TestContainersConfig {
 		return TestContainersConfig{}
```

Comment:
I'm going to merge this one, although I wonder if the struct we are returning here should consider the env variables too 🤔